plasma/plasma-mvp/morevp-simplified.md

```diff
@@ -0,0 +1,87 @@
+1. `tx` is correctly formed and properly signed.
+2. None of the inputs to `tx` have already been spent in some transaction other than `tx`.
+3. `out` has not been spent.
+
+The first requirement in both games ensures that the transaction is correctly formed (input amounts are greater than or equal to outputs amounts, referencing valid inputs) and signatures over the inputs come from their correct respective owners.
+
+The second requirement has something to do with a concept we call transaction “canonicity.”
+Simply defined, a transaction is “canonical” if none of the transaction’s inputs have been previously spent.
+Let's suppose there exist two transactions `tx` and `other_tx` such that `tx` and `other_tx` share at least one input.
+If `tx` *is* included in the Plasma chain, then `tx` is not canonical if `other_tx` is included before `tx` (in an earlier block or in the same block and an earlier transaction index).
+If `tx` is *not* included in the Plasma chain, then `tx` is not canonical if `other_tx` exists.
+In this sense, the shared input is being double-spent in `tx` because it was already used in `other_tx`. 
+`tx` is canonical if a transaction `other_tx` that would make `tx` not canonical by the above rules does not exist.
+
+We say that a transaction is *fully valid* if it's correctly formed, properly signed, and canonical.
+
+If a transaction is correctly formed and signed, but the transaction *is not* canonical, then, as shown above, some input to that transaction must have been double-spent.
+However, it may be the case that not *all* of the inputs to the transaction are double-spent - other than this transaction, these inputs are unspent.
+In this case, we should allow the owners of these inputs to retrieve their funds.
+The third requirement filters out any inputs that have been double-spent. 
+
+If a transaction is correctly formed and signed, and the transaction *is* canonical, then this is a fully valid transaction.
+In this case, the outputs to this transaction should receive the resulting funds, as is expected.
+Here, the third requirement then simply ensures that only *unspent* outputs will be able to withdraw funds from the contract.
+
+This very basic exit game simply ensures that the “true” owners of money within any transaction will be able to exit.
+
+### Priority
+
+Note, however, that the above exit game fails to provide safety if exit priority is still determined by the position of the transaction input or output in the Plasma chain (“output-age priority”). 
+
+To illustrate, imagine the following scenario:
+
+1. Alice sends transaction `tx` to Bob.
+2. The operator creates a block with invalid transaction `inv_tx`, and includes `tx` after `inv_tx`.
+3. Bob starts the exit game with `tx`.
+4. The operator starts the exit game with `inv_tx`.
+
+Because both `tx` and `inv_tx` appear to be properly formed and signed, both Bob and the operator can prove (1).
```

Comment:
which 1/2/3 are you referring to - I'm guessing that one of the list at the beginning, but which?

Reply:
Both are outputs playing the exit game, so both are referencing the requirements for owners of outputs. Will clarify.